tests/test_cmd_signature.py

```diff
@@ -90,6 +90,25 @@ def test_sig_merge_1_multisig(c):
     assert actual_merge_sig.minhash == test_merge_sig.minhash
 
 
+@utils.in_tempdir
+def test_sig_merge_1_name(c):
+    # check name arg
+    sig2 = utils.get_test_data('2.fa.sig')
+    sig63 = utils.get_test_data('63.fa.sig')
+
+    assignedSigName = 'SIG_NAME'
```

Comment:
for future reference, we _typically_ use non-CamelCase variables for things that aren't class names. So it's `MinHash` and `SourmashSignature` but `load_signature`. In this case, I would suggest `assigned_sig_name`. As this is a test, however, there's no need to change it, this is just an FYI!